### PR TITLE
pie-docs: Added elevation global tokens.

### PR DESCRIPTION
**Pull request by kevinrodrigues**
[Added] - Elevation global token page.

### CODE REVIEW

File: apps/pie-docs/src/tokensMetadata.json [hunk 1]
```diff
@@ -1501,6 +1501,52 @@
     }
   },
   "elevation": {
+    "global": {
+      "box-shadow-00": {
+        "displayName": "Box shadow 00",
```
Comment (kevinrodrigues) [inline apps/pie-docs/src/tokensMetadata.json:1506]: Temp value for 00 for now. Discussed with Jordi.

File: apps/pie-docs/src/tokensMetadata.json [hunk 2]
```diff
@@ -1501,6 +1501,52 @@
...
+        "description": "0,0,0,0 rgba(0,0,0,.00)<br /> 0,0,0,0 rgba(0,0,0,.00)<br /> 0,0,0,0 rgba(0,0,0,.00)"
```
Comment (jamieomaguire) [inline apps/pie-docs/src/tokensMetadata.json:1507]: is `.00` the correct value?

File: apps/pie-docs/src/tokensMetadata.json [hunk 3]
```diff
@@ -1501,6 +1501,52 @@
...
+      },
+      "box-shadow-01": {
+        "displayName": "Box shadow 01",
+        "description": "0,2,2,0 rgba(27,35,36,.03)<br /> 0,3,1,-2 rgba(27,35,36,.07)<br /> 0,1,5,0 rgba(27,35,36,.06)"
+      },
+      "box-shadow-02": {
+        "displayName": "Box shadow 02",
+        "description": "0,4,6,0 rgba(27,35,36,.02)<br /> 0,2,12,-2 rgba(27,35,36,.08)<br /> 0,3,6,0rgba(27,35,36,.06)"
```
Comment (jamieomaguire) [inline apps/pie-docs/src/tokensMetadata.json:1515]:
```suggestion
        "description": "0,4,6,0 rgba(27,35,36,.02)<br /> 0,2,12,-2 rgba(27,35,36,.08)<br /> 0,3,6,0 rgba(27,35,36,.06)"
```

Comment (jamieomaguire) [reply apps/pie-docs/src/tokensMetadata.json:1515]: might be worth checking for more (I will too) - they're present in Figma too

File: apps/pie-docs/src/tokensMetadata.json [hunk 4]
```diff
@@ -1501,6 +1501,52 @@
...
+        "displayName": "Box shadow 03",
+        "description": "0,2,6,0 rgba(27,35,36,.04)<br /> 0,8,12,-2 rgba(27,35,36,.06)<br /> 0,4,6,0 rgba(27,35,36,.04)"
+      },
+      "box-shadow-04": {
+        "displayName": "Box shadow 04",
+        "description": "0,8,8,0 rgba(27,35,36,.04)<br /> 0,8,20,-3 rgba(27,35,36,.10)<br /> 0,4,8,-2 rgba(27,35,36,.06)"
+      },
+      "box-shadow-05": {
+        "displayName": "Box shadow 05",
+        "description": "0,-4,6,0 rgba(27,35,36,.02)<br /> 0,-2,12,-2 rgba(27,35,36,.08)<br /> 0,-3,6,0rgba(27,35,36,.06)"
```
Comment (jamieomaguire) [inline apps/pie-docs/src/tokensMetadata.json:1527]:
```suggestion
        "description": "0,-4,6,0 rgba(27,35,36,.02)<br /> 0,-2,12,-2 rgba(27,35,36,.08)<br /> 0,-3,6,0 rgba(27,35,36,.06)"
```

File: apps/pie-docs/src/tokensMetadata.json [hunk 5]
```diff
@@ -1501,6 +1501,52 @@
...
+      },
+      "box-shadow-06": {
+        "displayName": "Box shadow 06",
+        "description": "0,2,2,0 rgba(27,35,36,.12)<br /> 0,3,1,-2 rgba(27,35,36,.28)<br /> 0,1,5,0 rgba(27,35,36,.24)"
+      },
+      "box-shadow-07": {
+        "displayName": "Box shadow 07",
+        "description": "0,4,6,0 rgba(27,35,36,.08)<br /> 0,2,12,-2 rgba(27,35,36,.32)<br /> 0,3,6,0rgba(27,35,36,.24)"
```
Comment (jamieomaguire) [inline apps/pie-docs/src/tokensMetadata.json:1535]:
```suggestion
        "description": "0,4,6,0 rgba(27,35,36,.08)<br /> 0,2,12,-2 rgba(27,35,36,.32)<br /> 0,3,6,0 rgba(27,35,36,.24)"
```

File: apps/pie-docs/src/tokensMetadata.json [hunk 6]
```diff
@@ -1501,6 +1501,52 @@
...
+        "displayName": "Box shadow 08",
+        "description": "0,2,6,0 rgba(27,35,36,.12)<br /> 0,8,12,-2 rgba(27,35,36,.24)<br /> 0,4,6,0 rgba(27,35,36,.16)"
+      },
+      "box-shadow-09": {
+        "displayName": "Box shadow 09",
+        "description": "0,8,8,0 rgba(27,35,36,.16)<br /> 0,3,20,-3 rgba(27,35,36,.40)<br /> 0,4,8,-2 rgba(27,35,36,.24)"
+      },
+      "box-shadow-10": {
+        "displayName": "Box shadow 10",
+        "description": "0,-4,6,0 rgba(27,35,36,.08)<br /> 0,-2,12,-2 rgba(27,35,36,.32)<br /> 0,-3,6,0rgba(27,35,36,.24)"
```
Comment (jamieomaguire) [inline apps/pie-docs/src/tokensMetadata.json:1547]:
```suggestion
        "description": "0,-4,6,0 rgba(27,35,36,.08)<br /> 0,-2,12,-2 rgba(27,35,36,.32)<br /> 0,-3,6,0 rgba(27,35,36,.24)"
```

**Comment by jamieomaguire** [comment]
Awesome stuff! Just a few minor comments :)

File: apps/pie-docs/src/_11ty/shortcodes/tokensTable/index.js [hunk 7]
```diff
@@ -215,7 +215,8 @@ const buildUncategorisedLists = ({
         tokenScssName: tokenTypeMetadata[token[0]]?.scssName ?? createScssTokenName(token[0], tokenType, path),
         tokenDisplayName: tokenTypeMetadata[token[0]]?.displayName ?? createTokenDisplayName(token[0], tokenType),
         tokenType,
-        tokenMetadata: tokenTypeMetadata[token[0]]
+        tokenMetadata: tokenTypeMetadata[token[0]],
+        path
```
[the remark this answers is not in the document]
Comment (kevinrodrigues) [reply apps/pie-docs/src/_11ty/shortcodes/tokensTable/index.js:219]: Yup it needs it and is part of the parameter function too.

File: apps/pie-docs/src/tokensMetadata.json (hunk 4 above)
Comment (kevinrodrigues) [reply apps/pie-docs/src/tokensMetadata.json]: Fixed

File: apps/pie-docs/src/tokensMetadata.json (hunk 2 above)
Comment (kevinrodrigues) [reply apps/pie-docs/src/tokensMetadata.json]: fixed.

**Review by jamieomaguire — APPROVED** [review]

File: apps/pie-docs/src/tokensMetadata.json [hunk 8]
```diff
@@ -1501,6 +1501,52 @@
     }
   },
   "elevation": {
+    "global": {
+      "box-shadow-00": {
+        "displayName": "Box shadow 00",
+        "description": "0,0,0,0 rgba(0,0,0,0)<br /> 0,0,0,0 rgba(0,0,0,0)<br /> 0,0,0,0 rgba(0,0,0,0)"
+      },
+      "box-shadow-01": {
+        "displayName": "Box shadow 01",
```
Comment (raoufswe) [inline apps/pie-docs/src/tokensMetadata.json:1510]: the display names are Elevation 0x in designs, are you sure we should use box shadow 0x instead?

**Review by raoufswe — APPROVED** [review]

**Review by LTurns — APPROVED** [review]

**Pull request merged**